cookbook/catalog/common/index.rst

```diff
@@ -1,4 +1,4 @@
-How to Manipulate the Non-Product Objects
+How to Manipulate Non-Product Objects
 =========================================
```

Comment:
you can remove few "="  with restructured text, you should have same number of underlines than characters in the title